src/report.py

```diff
@@ -0,0 +1,98 @@
+import csv
+from typing import List, Dict, Tuple
+
+
+def read_portfolio(filename: str) -> List[Dict]:
```

Comment:
Same remark here for the naming, it should be `file_path`
```suggestion
def read_portfolio(file_path: str) -> List[Dict]:
```

Also regarding the static typing, when you have a dict you can specify the type of the keys and values. From what I understand the keys will be string and the values can be string / int / float so you can adapt the typing to
`List[Dict[str, Union[str, int, float]]]`. This typing will provide more information to a new user than a simply `List[Dict]`